src/components/Board/index.spec.js

```diff
@@ -657,7 +658,8 @@ describe('<Board />', () => {
             expect(onCardDragEnd).toHaveBeenCalledWith(
               expectedBoard,
               { fromPosition: 0, fromLaneId: 1 },
-              { toPosition: 1, toLaneId: 1 }
+              { toPosition: 1, toLaneId: 1 },
+              { description: 'Card content', id: 1, title: 'Card title 1' }
```

Comment:
I think we should pass the card as the second argument. It doesn't make sense to keep the coordinates between the models.